iam.tf

```diff
@@ -113,6 +113,13 @@ resource "aws_iam_policy" "write_to_cw" {
       "Resource": [
           "*"
       ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+         "ec2:ModifyInstanceMetadataOptions"
```

Comment:
If there is only a single Action, it does not need to be specified as a list.

Reply:
Hi @ianrelf 
thanks. I have kept it as a list, where there will be more scrutinity added in the future. As of now, it allows to modify instance metadata across all instances in the VPC. I have made this change in number of places, will certainly need a list to add more policy in the near future.

Reply:
Let's add a list when we need one.